CHANGELOG.md

```diff
@@ -1485,7 +1485,7 @@ more information see https://openpolicyagent.org/docs/latest/privacy/.
 #### New `opa build` command
 
 The `opa build` command can now be used to package OPA policy and data files
-into [bundles](https://www.openpolicyagent.org/docs/latest/management/#bundles)
+into [bundles](https://www.openpolicyagent.org/docs/bundles.html)
```

Comment:
This one doesn't look quite right 🤔 This one would be redirected to `/docs/latest/bundles` via
https://github.com/open-policy-agent/opa/blob/main/docs/website/layouts/index.redirects#L33, which redirects to `/docs/latest/management-bundles`, so can we just link there directly?